preprocessing/nextclade/tests/test_processing_functions.py

```diff
@@ -661,5 +667,49 @@ def test_parse_date_into_range() -> None:
     ), "dateRangeLower: empty date should be returned as None."
 
 
+@pytest.mark.parametrize(
+    "case_dir",
+    ["cchf_1", "west_nile_1"],
+    ids=["cchf_1", "west_nile_1"],
+)
+def test_create_flatfile(config: Config, case_dir):
```

Comment:
maybe add a comment that this test requires that there is only one sample (single or multi-segmented) in the files

Reply:
Ah, makes sense; now that you mention it I can see how there could be multiple different ones in the same file.